routes/views/job.js

```diff
@@ -8,34 +8,35 @@ exports = module.exports = function (req, res) {
 	// Set locals
 	locals.section = 'jobs';
 	locals.filters = {
-		post: req.params.post,
+		job: req.params.job,
 	};
 	locals.data = {
-		posts: [],
+		jobss: [],
 	};
 
-	// Load the current post
+	// Load the current job
 	view.on('init', function (next) {
 
 		var q = keystone.list('Job').model.findOne({
 			state: 'published',
-			slug: locals.filters.post,
+			slug: locals.filters.job,
 		}).populate('author categories');
 
 		q.exec(function (err, result) {
-			locals.data.post = result;
+			locals.data.job = result;
 			next(err);
 		});
 
 	});
 
-	// Load other posts
+	// Load other jobs
 	view.on('init', function (next) {
 
 		var q = keystone.list('Job').model.find().where('state', 'published').sort('-publishedDate').populate('author').limit('4');
 
 		q.exec(function (err, results) {
-			locals.data.posts = results;
+			// locals.data.posts = results;
```

Comment:
@maryam-olabisi Please remove the commented out code here

Reply:
Done